docs/julia.rst

```diff
@@ -0,0 +1,399 @@
++---------------------------------------------------------------------------------------------------------------+
+| mulmodu256(x:u256, y:u256, m:u256) -> z:u256| (x * y) % m with arbitrary precision arithmetics                |
++---------------------------------------------------------------------------------------------------------------+
+| ltu256(x:u256, y:u256) -> z:bool            | 1 if x < y, 0 otherwise                                         |
++---------------------------------------------------------------------------------------------------------------+
+| gtu256(x:u256, y:u256) -> z:bool            | 1 if x > y, 0 otherwise                                         |
++---------------------------------------------------------------------------------------------------------------+
+| sltu256(x:u256, y:u256) -> z:bool           | 1 if x < y, 0 otherwise, for signed numbers in two's complement |
++---------------------------------------------------------------------------------------------------------------+
+| sgtu256(x:u256, y:u256) -> z:bool           | 1 if x > y, 0 otherwise, for signed numbers in two's complement |
++---------------------------------------------------------------------------------------------------------------+
+| equ256(x:u256, y:u256) -> z:bool            | 1 if x == y, 0 otherwise                                        |
++---------------------------------------------------------------------------------------------------------------+
+| notu256(x:u256) -> z:u256                   | ~x, every bit of x is negated                                   |
++---------------------------------------------------------------------------------------------------------------+
+| andu256(x:u256, y:u256) -> z:u256           | bitwise and of x and y                                          |
++---------------------------------------------------------------------------------------------------------------+
+| oru256(x:u256, y:u256) -> z:u256            | bitwise or of x and y                                           |
++---------------------------------------------------------------------------------------------------------------+
+| xoru256(x:u256, y:u256) -> z:u256           | bitwise xor of x and y                                          |
++---------------------------------------------------------------------------------------------------------------+
+| shlu256(x:u256, y:u256) -> z:u256           | logical left shift of x by y                                    |
++---------------------------------------------------------------------------------------------------------------+
+| shru256(x:u256, y:u256) -> z:u256           | logical right shift of x by y                                   |
++---------------------------------------------------------------------------------------------------------------+
+| saru256(x:u256, y:u256) -> z:u256           | arithmetic right shift of x by y                                |
```

Comment:
Perhaps it would be safer to only allow bit operations on a bit-type that has no direct meaning as a number.